frontend/src/scenes/insights/insightLogic.tsx

```diff
@@ -87,7 +89,12 @@ export const insightLogic = kea<insightLogicType>([
             insight,
             options,
         }),
-        saveAsNamingSuccess: (name: string, redirectToViewMode?: boolean) => ({ name, redirectToViewMode }),
+        saveAs: (query: Node, redirectToViewMode?: boolean) => ({ query, redirectToViewMode }),
+        saveAsConfirmation: (name: string, query: Node, redirectToViewMode?: boolean) => ({
```

Comment:
Renaming from `saveAsSuccess` top `saveAsConfirmation`. This action is triggered after selecting the desired name in a LemonModal dialog. The `Success` suffix is usually reserved for kea-loader generated actions.